x/superfluid/keeper/intermediary_account_test.go

```diff
@@ -26,6 +26,18 @@ func (suite *KeeperTestSuite) TestIntermediaryAccountCreation() {
 			2,
 			[]superfluidDelegation{{0, 0, 0, 1000000}, {1, 1, 0, 1000000}},
 		},
+		{
```

Comment:
nit: can we add details on what we're testing in the name or comments for the expected behavior when the validator is unbonded or unbonding?